docs/UserGuide.md

```diff
@@ -226,6 +226,25 @@ Scheduled meeting with John Doe from 23-02-2024 16:00 to 23-02-2024 17:00
 Error: cannot schedule more than 1 meeting with a contact!
 ```
 
+### Rescheduling a meeting : `reschedule`
+
+Allows the currently scheduled meeting details to be edited.
+
+Format: `reschedule INDEX [s/START_DATETIME] [e/END_DATETIME] [mr/MEETING_REMARK]`
+
+* The index refers to the index number shown in the displayed person list.
+* The index **must be a positive integer** 1, 2, 3, …​
+* Specified person must have a meeting scheduled.
+* At least one of the optional fields must be provided.
+* Input values will overwrite existing value that were assigned to the specified field.
+* `START_DATETIME` must be after the system's current datetime.
+* `END_DATETIME` must be after `START_DATETIME`
+* `MEETING_REMARK` has the same constraints as `REMARK` as stated above.
```

Comment:
I think REMARK does not have a length limit like MEETING_REMARK? I think we should include the character limit or add an additional MEETING_REMARK constraints above that includes the length character limit and allowed set of characters?

Reply:
Added MEETING_REMARK format to the top of UG